File: DSA/Data/reverse/reversearray.py
# def reversearray(arr) :
#     start = 0 
#     end = 4
#     while start<end :
#         arr[start], arr[end] = arr[end], arr[start]
#         start += 1
#         end -= 1

# arr = [1,2,3,4,5]
# reversearray(arr)
# print(arr)


def reverseList(A):
  print( A[::-1])
     
# Driver function to test above function
A = [1, 2, 3, 4, 5, 6]
print(A)
print("Reversed list is")
reverseList(A) 

File: DSA/Data/reverse/reversearray.js

function rvereseArray(arr, start, end) {
    while (start < end) {
        var temp = arr[start];
        arr[start] = arr[end];
        arr[end] = temp;
        start++;
        end--;
    }
}

function printArray(arr, size) {
    for (var i = 0; i < size; i++) {
        console.log(arr[i]);
    }
}

/* Driver function to test above functions */
var arr = [1, 2, 3, 4, 5, 6];
var n = 6;
// To print original array
printArray(arr, n);

// Function calling
rvereseArray(arr, 0, n - 1);

console.log("Reversed array is");

// To print the Reversed array
printArray(arr, n);
